include/libkernelflinger/log.h

```diff
@@ -38,9 +38,13 @@
 #include <vars.h>
 
 EFI_STATUS log_flush_to_var(BOOLEAN nonvol);
+EFI_FILE *efi_log_file;
+EFI_FILE *g_root;
 
 void log(const CHAR16 *fmt, ...);
 void vlog(const CHAR16 *fmt, va_list args);
+EFI_FILE *log_open_output_file(const CHAR16 *filename);
+void log_close_output_file();
```

Comment:
**Coding style violation**: This is not a very hard rule since it does not make any real difference regarding the maintenance or software quality but most of the time in the legacy code, prototype with no argument are declared as follow:
`void log_close_output_file(void);`

Reply:
Already modified.